src/components/App/App.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable max-len */
+import React from 'react';
+import './App.scss';
+import Carousel from '../Carousel/Carousel';
+
+interface AppState {
+  images: string[],
+  step: number,
+  frameSize: number,
+  itemWidth: number,
+  animationDuration: number,
+  infinite: boolean
+}
+
+class App extends React.Component<{}, AppState> {
+  state: Readonly<AppState> = {
+    images: [
+      './img/1.png',
```

Comment:
I would rather suggest moving this whole images array above before `class App` and just storing it in a variable like `imagesData` and use it here to improve readability:

```suggestion
      'state: Readonly<AppState> = {
          images: imagesData',
          step: 3,
```